py/objarray.c

```diff
@@ -548,7 +562,11 @@ const mp_obj_type_t mp_type_bytearray = {
     .binary_op = array_binary_op,
     .subscr = array_subscr,
     .buffer_p = { .get_buffer = array_get_buffer },
+#if MICROPY_CPYTHON_COMPAT    
```

Comment:
The usual comment: please indent `#if`. (Yeah, I recently work on a project which doesn't follow this obvious rule, resulting in very ugly looking code, so am trained to recognize even minor cases of it.)

Reply:
Yes switching between projects and different styles makes me forgetting to do this correctly :[